src/logical.jl

```diff
@@ -24,7 +24,7 @@ rather ther function `Eq` is used. Here we use the unicode
 * `>=` is `\ge<tab>`
 * `>` is `\gg<tab>`
 
-"""
+=#
```

Comment:
I have commented this out as this do not document anything and is disallow on nigthlies see: https://github.com/JuliaLang/julia/pull/13708.